admin_guide/encrypting_data.adoc

```diff
@@ -0,0 +1,261 @@
+[[admin-guide-encrypting-data-rest]]
+= Encrypting Data at Rest
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[encrypting-data-overview]]
+== Overview
+
+This topic shows how to enable and configure encryption of secret data at rest.
+
+[[encrypting-data-configuration]]
+== Configuration and Determining Whether Encryption at Rest is Already Enabled
```

Comment:
@tiran This violates our docs guide: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#topic-titles-and-section-headings